src/quickstart/provider/azure-acs/install-azure-acs.md

```diff
@@ -1,85 +1,14 @@
 # Install Deis Workflow on Azure Container Service
 
-## Check Your Setup
-
-First check that the `helm` command is available and the version is v2.0.0 or newer.
-
-```
-$ helm version
-Client: &version.Version{SemVer:"v2.0.0", GitCommit:"51bdad42756dfaf3234f53ef3d3cb6bcd94144c2", GitTreeState:"clean"}
-Server: &version.Version{SemVer:"v2.0.0", GitCommit:"51bdad42756dfaf3234f53ef3d3cb6bcd94144c2", GitTreeState:"clean"}
-```
-
-Finally, intialize Helm:
-```
-helm init
-```
-
-Ensure the `kubectl` client is installed and can connect to your Kubernetes cluster.
-
-## Add the Deis Chart Repository
-
-The Deis Chart Repository contains everything needed to install Deis Workflow onto a Kubernetes cluster, with a single `helm install deis/workflow --namespace deis` command.
-
-Add this repository to Helm:
-
-```
-$ helm repo add deis https://charts.deis.com/workflow
-```
 
 ## Create New Azure Storage Account
 
 It is recommended to use a dedicated storage account for the operational aspects of Workflow, which includes storing slug and container images, database backups, and disaster recovery. This storage account is passed as parameters during the `helm install` command in the next step. Replace the `SA_NAME` variable with a unique name for your storage account and execute these commands.
+
 ```
 $ export SA_NAME=YourGlobalUniqueName
 $ az storage account create -n $SA_NAME -l $DC_LOCATION -g $RG_NAME --sku Standard_LRS
 $ export SA_KEY=`az storage account keys list -n $SA_NAME -g RG_NAME --query keys[0].value --output tsv`
-
-```
-
-## Install Deis Workflow
-
-Now that Helm is installed and the repository has been added, install Workflow by running:
-
-```
-$ helm install deis/workflow --namespace=deis --set controller.k8s_api_verify_tls=false,global.storage=azure,azure.accountname=$SA_NAME,azure.accountkey=$SA_KEY,azure.registry_container=registry,azure.database_container=database,azure.builder_container=builder
```

Comment:
Ah yes, the custom Azure setup. Let's leave the Azure docs separate for now until we can get deis/controller#1193 out the door. Nice catch 👍